src/api/app/assets/stylesheets/webui2/collapsible-text.scss

```diff
@@ -1,8 +1,8 @@
 @mixin collapsible($maxHeight: 4.5em) {
   .obs-collapsible-textbox {
-    word-break: break-word;
     padding-right: 1em;
-    text-align: justify;
+    hyphens: auto;
+    max-width: 600px;
```

Comment:
is that a bad pattern to add a comment here explaining why we went for 600px? Our reasoning was to be around 65-80 chars long.

Reply:
I don't see how putting a comment explaining something would be a bad pattern... For me, you're good to go and write it just there

Reply:
hm maybe I just didn't express myself well.. yes comments are not bad, but we don't do it in css so I'm not sure if just this arbitrary value among all  other arbitrary values in our CSS should be commented.

Reply:
We could use a Sass variable for this and it would have 2 benefits:

- It documents itself (if the variable has a name like: `80_characters_width`)
- We are consistent in all places where we use the variable

Reply:
I liked it. Just the variable name should be something else, since it cannot start with a number.

Reply:
`eighty_characters_width` maybe?

Reply:
Actually with this calculator http://www.unitconversion.org/typography/characters-x-to-pixels-x-conversion.html, 600px are 75 chars :)

Reply:
maybe it varies with font-size, or font-family?

Reply:
> is that a bad pattern to add a comment here explaining why we went for 600px?

No it's cool. Better would be to also have a single commit with a good commit description that you can blame :cowboy_hat_face: